src/rpc/protocol.h

```diff
@@ -48,7 +48,7 @@ enum RPCErrorCode
     RPC_VERIFY_ALREADY_IN_CHAIN     = -27, //!< Transaction already in chain
     RPC_IN_WARMUP                   = -28, //!< Client still warming up
     RPC_METHOD_DEPRECATED           = -32, //!< RPC method is deprecated
-    RPC_PLATFORM_RESTRICTION        = -33, //!< This RPC command cannot be run by platform-user
+    RPC_RESERVED_UNUSED             = -33, //!< This RPC code has been used in past, let it be reserved
```

Comment:
let's maybe drop since it actually conflicts with `RPC_CLIENT_MEMPOOL_DISABLED = -33, //!< No mempool instance found`